aws/resource_aws_ses_identity_notification_test.go

```diff
@@ -0,0 +1,98 @@
+package aws
+
+import (
+	"fmt"
+	"testing"
+
+	"github.com/aws/aws-sdk-go/aws"
+	"github.com/aws/aws-sdk-go/service/ses"
+	"github.com/hashicorp/terraform/helper/acctest"
+	"github.com/hashicorp/terraform/helper/resource"
+	"github.com/hashicorp/terraform/terraform"
+)
+
+func TestAccAwsSESIdentityNotification_basic(t *testing.T) {
+	domain := fmt.Sprintf(
+		"%s.terraformtesting.com",
+		acctest.RandStringFromCharSet(10, acctest.CharSetAlphaNum))
+
+	resource.Test(t, resource.TestCase{
+		PreCheck: func() {
+			testAccPreCheck(t)
+		},
+		Providers:    testAccProviders,
+		CheckDestroy: testAccCheckAwsSESIdentityNotificationDestroy,
+		Steps: []resource.TestStep{
```

Comment:
We should have a `TestStep` that attempts to update `topic_arn`

Reply:
@bflad I fixed everything but this. I do not know how :-)

For some reason I managed to pollute this PR with the history of terraform... Unsure how I undo that. I could create a new PR - it seems that would only attach the valid commits?

Reply:
@bflad added a case where I push an updated arn. Hope this is the correct implementation.